src/Runtime.php

```diff
@@ -130,6 +130,14 @@ public function getBinary(): string
         return self::$binary;
     }
 
+    /**
+     * Returns the path to the binary of the current runtime.
+     */
+    public function getBinary(): string
+    {
+        return escapeshellarg($this->getRawBinary());
```

Comment:
after this change even the hardcoded `'php'` (line 126) is passed thru `escapeshellarg`, which wasn't the case before.

tested locally on my macos, that this still works as expected